src/express/commands/send-stake-update.js

```diff
@@ -23,6 +24,8 @@ export async function sendStakeUpdateEvent(validatorID) {
 
   const doc = await loadDevnetConfig(devnetType)
   let machine0
+  const fundingKey =
```

Comment:
Do we need to have an hardcoded `fundingKey`?
This step is not clear to me.
I see you mentioned a problem with the account management (and mnemonic) but I have two questions:
- will the fundingKey always be the same in each and every setup?
- can we simply fetch this account with a foundry/anvil command? That must exist somewhere